src/components/Table/Table/__tests__/table.jest.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import { render, cleanup } from "@testing-library/react";
+import TableCell from "../../TableCell/TableCell";
+import Table from "../Table";
+import TableBody from "../../TableBody/TableBody";
+import TableRow from "../../TableRow/TableRow";
+import TableHeaderCell, { ITableHeaderCellProps } from "../../TableHeaderCell/TableHeaderCell";
+import TableHeader from "../../TableHeader/TableHeader";
+
+interface TableNode {
```

Comment:
I didn't see any tests related to virtualized table or error state, let's talk about it in our meeting?

Reply:
Also let's talk about if we want to test somehow the --table-grid-template-columns mechanism

Reply:
Agreed, I'll add a test for that